third_party/istio-1.0.0/README.md

```diff
@@ -0,0 +1,15 @@
+This istio.yaml file is is generated by running
+```
+./download-istio.sh
+```
+
+The Helm options we used are:
+
+1. `sidecarInjectorWebhook.enabled=true` & `sidecarInjectorWebhook.enableNamespacesByDefault=true`: We allow sidecar injection on all namespaces.
+2. `global.proxy.autoInject=disabled`: However, only apply sidecar injection for Pods annotated with `istio.sidecar.inject=true`, and not as a default.
+3. `prometheus.enabled=false`: Disable Prometheus by default.
+
+Our goal here is to allow sidecar injection for Pods created by Knative, and
+nothing else.  This template is used in integration tests and also released as
+an Istio-one-line-installation so that our users don't have to go through a lot
+of steps to install Istio.
```

Comment:
It would be good to make note of whether Istio installed as configured here is a production-ready configuration or more of a "kick-the-tires" configuration. I honestly do not know, but it would be good to call it out.

Reply:
This comment is non-blocking, btw.